test/test_browser.py

```diff
@@ -831,13 +831,13 @@ def test_sdl_stb_image_cleanup(self):
     shutil.copyfile(test_file('screenshot.jpg'), 'screenshot.not')
     self.btest_exit('test_sdl_stb_image_cleanup.c', args=['-sSTB_IMAGE', '--preload-file', 'screenshot.not', '-lSDL', '-lGL', '--memoryprofiler'])
 
-  def test_sdl_canvas(self):
-    self.btest_exit('test_sdl_canvas.c', args=['-sLEGACY_GL_EMULATION', '-lSDL', '-lGL'])
-    # some extra coverage
-    self.clear()
-    self.btest_exit('test_sdl_canvas.c', args=['-sLEGACY_GL_EMULATION', '-O0', '-sSAFE_HEAP', '-lSDL', '-lGL'])
-    self.clear()
-    self.btest_exit('test_sdl_canvas.c', args=['-sLEGACY_GL_EMULATION', '-O2', '-sSAFE_HEAP', '-lSDL', '-lGL'])
+  @parameterized({
+    '': ([],),
+    '_safe_heap': (['-sSAFE_HEAP'],),
+    '_safe_heap_O2': (['-sSAFE_HEAP', '-O2'],),
```

Comment:
We don't need the `_` prefixes in the names, I think? We don't add them anywhere else iirc.

Reply:
oh!  well spotted